### PR TITLE
Adjust light temperature

### PR DESCRIPTION
**Pull request by lessthnthree**
## About The Pull Request

Minor lighting adjustments based on feedback, more neutral temperatures.

Regular lighting less blue hues
Nightshift lighting less pink hues
Fire alarm more red, less pink hues
Low power less UV blacklight, more dark orange
Reduces delay in turning on lights by half

## How This Contributes To The Skyrat Roleplay Experience

Lights don't look as harsh.

## Proof of Testing
<details>
<summary>Screenshots/Videos</summary>
Before/After

image

image

Before/After

image

image

Before/After

image

image
  
</details>

## Changelog

:cl: LT3
code: Minor lighting adjustments: more neutral, less blue and pink hues
code: Lights turn on faster
/:cl:

### CODE REVIEW

File: code/modules/power/lighting/light.dm [hunk 1]
```diff
@@ -178,8 +178,8 @@
 
 
 //SKYRAT EDIT ADDITION BEGIN - AESTHETICS
-#define LIGHT_ON_DELAY_UPPER (4 SECONDS)
-#define LIGHT_ON_DELAY_LOWER (1 SECONDS)
+#define LIGHT_ON_DELAY_UPPER (2 SECONDS)
+#define LIGHT_ON_DELAY_LOWER (0.25 SECONDS)
```
Comment (GoldenAlpharex) [inline code/modules/power/lighting/light.dm:182]: This is an undocumented change.

File: modular_skyrat/modules/aesthetics/lights/code/lighting.dm [hunk 2]
```diff
@@ -10,13 +10,13 @@
...
-	fire_colour = "#D47F9B"
-	bulb_colour = LIGHT_COLOR_FAINT_BLUE
+	fire_colour = COLOR_FIRE_LIGHT_RED
+	bulb_colour = COLOR_OFF_WHITE
 	bulb_power = 1.15
 	nightshift_light_color = null // Let the dynamic night shift color code handle this.
-	bulb_low_power_colour = LIGHT_COLOR_DARK_BLUE
-	bulb_low_power_brightness_mul = 0.4
-	bulb_low_power_pow_min = 0.4
+	bulb_low_power_colour = LIGHT_COLOR_BROWN
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/aesthetics/lights/code/lighting.dm:17]: Would be nice to actually see what those look like now, there doesn't seem to be much in terms of low power lights in your examples. I'm not certain that brown will necessarily look the best either.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

File: code/modules/power/lighting/light.dm (hunk 1 above)
Comment (lessthnthree) [reply code/modules/power/lighting/light.dm:182]: Whoop, I meant to document that.

File: modular_skyrat/modules/aesthetics/lights/code/lighting.dm (hunk 2 above)
Comment (lessthnthree) [reply modular_skyrat/modules/aesthetics/lights/code/lighting.dm:17]: Low power mode is when there is no power, running on the backup cell. Going for a dim emergency light look. Warm temp, some red, but not as red as the fire alarm/emergency lighting.

image

Comment (lessthnthree) [reply modular_skyrat/modules/aesthetics/lights/code/lighting.dm:17]: Current low power:

image

Comment (lessthnthree) [reply modular_skyrat/modules/aesthetics/lights/code/lighting.dm:17]: Fire alarm after:

image

Fire alarm current:

image

**Review by GoldenAlpharex — APPROVED** [review]

**Pull request merged**